common/utils/src/main/resources/error/error-classes.json

```
@@ -97,6 +97,12 @@
       "The method <methodName> can not be called on streaming Dataset/DataFrame."
     ]
   },
+  "CANNOT_ALTER_PARTITION_COLUMN" : {
+    "message" : [
+      "ALTER TABLE (ALTER|CHANGE) COLUMN is not supported for partition columns, but found the partition column <columnName> in the table <tableName>"
```

nit: for consistency to other error message formats in the file:
```suggestion
      "ALTER TABLE (ALTER|CHANGE) COLUMN is not supported for partition columns, but found the partition column <columnName> in the table <tableName>."
```